src/it/junit/junit-rerun/src/test/resources/features/feature2.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Feature1
```

Comment:
should we say "Feature: Feature2" here? Since the file is called feature2.feature.